### PR TITLE
The decisiontree-selected hazardset source is displayed under the map

### PR DESCRIPTION
**Pull request by fvanderbiest**
This is a proposal for https://github.com/GFDRR/thinkhazard/issues/40

Here's a screenshot taken during tests:
capture du 2015-12-14 14 55 33
(in which the hazardset is named "test")

Also, tests had to be updated as a result of https://github.com/GFDRR/thinkhazard_common/pull/9

### CODE REVIEW

**Comment by fvanderbiest** [comment]
DO not merge for now, I have an eror : ```InvalidRequestError: Could not find a FROM clause to join from.  Tried joining to <class 'thinkhazard_common.models.HazardType'>, but got: Can't find any foreign key relationships between 'Join object on administrativedivision(47872122408464) and rel_hazardcategory_administrativedivision(47872122272016)' and 'enum_hazardtype'.```

**Comment by fvanderbiest** [comment]
Still TBD:
 * sources as a list, e.g.: 'a, b, c.' for admin level 0 and 1
 * geonode's ```distribution_url``` as hyperlink on the data source name

**Comment by fvanderbiest** [comment]
Seems like this PR is not responsible of the above error.

Complete stack trace is:
```
2016-01-15 10:01:17,619 ERROR [pyramid_debugtoolbar][waitress] Exception at http://0.0.0.0:6543/report/31/FL.json?resolution=0.004665345964671402
traceback url: http://0.0.0.0:6543/_debug_toolbar/exception?token=ca5f3c34ee372a9c0898&tb=46948352445648
Traceback (most recent call last):
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/pyramid_debugtoolbar/toolbar.py", line 192, in toolbar_tween
    response = _handler(request)
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/pyramid_debugtoolbar/panels/performance.py", line 57, in resource_timer_handler
    result = handler(request)
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/pyramid/tweens.py", line 20, in excview_tween
    response = handler(request)
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/pyramid_tm/__init__.py", line 101, in tm_tween
    reraise(*exc_info)
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/pyramid_tm/__init__.py", line 83, in tm_tween
    response = handler(request)
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/pyramid/router.py", line 145, in handle_request
    view_name
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/pyramid/view.py", line 541, in _call_view
    response = view_callable(context, request)
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/pyramid/config/views.py", line 352, in rendered_view
    result = view(context, request)
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/pyramid/config/views.py", line 506, in _requestonly_view
    response = view(request)
  File "/home/fvanderbiest/workspace/thinkhazard/thinkhazard/views/report.py", line 204, in report_json
    .outerjoin(HazardType)\
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 1958, in outerjoin
    from_joinpoint=from_joinpoint)
  File "<string>", line 2, in _join
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/sqlalchemy/orm/base.py", line 201, in generate
    fn(self, *args[1:], **kw)
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2085, in _join
    outerjoin, create_aliases, prop)
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2153, in _join_left_to_right
    self._join_to_left(l_info, left, right, onclause, outerjoin)
  File "/home/fvanderbiest/workspace/thinkhazard/.build/venv/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2257, in _join_to_left
    "Tried joining to %s, but got: %s" % (right, ae))
InvalidRequestError: Could not find a FROM clause to join from.  Tried joining to <class 'thinkhazard_common.models.HazardType'>, but got: Can't find any foreign key relationships between 'Join object on administrativedivision(46948298628496) and rel_hazardcategory_administrativedivision(46948298487952)' and 'enum_hazardtype'.
```

**Comment by fvanderbiest** [comment]
Good to merge now, IMO. Please review.

File: thinkhazard/views/report.py [hunk 1]
```diff
@@ -159,7 +162,10 @@ def report(request):
     return {'hazards': hazard_types,
             'hazards_sorted': sorted(hazard_types,
                                      key=lambda a: a['hazardlevel'].order),
-            'hazard_category': hazard_category,
+            'hazard_category': association.hazardcategory
+            if association else '',
+            'source': association.source
+            if association else '',
```
Comment (pgiraud) [inline thinkhazard/views/report.py]: I'm suprised the indentation is correct here.

**Comment by pgiraud** [comment]
Please merge.

**Pull request merged**